tests/python_functional/src/syslog_ng/syslog_ng_executor.py

```diff
@@ -59,6 +59,24 @@ def run_process_with_valgrind(self):
             stderr_path=self.__instance_paths.get_stderr_path(),
         )
 
+    def run_process_with_strace(self):
+        strace_command_args = [
+            "strace",
+            "-s",
+            "4096",
+            "-tt",
+            "-T",
+            "-ff",
+            "-o",
+            self.__instance_paths.get_strace_log_path(),
```

Comment:
It would be better if the user could specify the parameters of `strace`. We can use a default of course, if not specified.

Reply:
As we discussed this PR only prepares the possibility of this feature.
After this PR next can resolve this feature also.